website/docs/reference/resource-configs/alias.md

```diff
@@ -40,3 +56,68 @@ seeds:
 ```
 
 </File>
+
+</File>
+</TabItem>
+
+<TabItem value="snapshot" label="Snapshots">
+
+Configure a seed's alias in your `dbt_project.yml` file. 
+
+For example, if you have a snapshot that represents `my_snapshot` and want to alias it as `updated_at_id`, you would alias like this:
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  - name: my_snapshot
+    config:
+      target_database: analytics
+      target_schema: finance
+      unique_key: id
+      strategy: timestamp
+      updated_at: updated_at
+      +alias: updated_at_id
+```
+
+This would return the name `analytics.finance.updated_at_id` in the database.
+
+</File>
+</TabItem>
+
+<TabItem value="test" label="Tests">
+
+Configure a test's alias in your `dbt_project.yml` file. 
```

Comment:
You can't create tests in a dbt_project.yml, this would be in a schema.yml file

Reply:
thank you! wait is schema.yml the same as properties.yml? https://docs.getdbt.com/reference/configs-and-properties#schemayml-files

Reply:
Yep!